xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -29,162 +29,266 @@
+   &microsd; card. This is the simplest and easiest method.
+  </para>
+  <para>
+   &suse; provides a pre-configured bootable image of &productname; for &rpi;
+   hardware. This comes with the Btrfs file system, with compression enabled
+   to improve performance and reduce wear on &microsd; media.
+  </para>
+  <para>
+   A &microsd; card with a minimum size of 8&nbsp;GB is recommended. Faster cards will
+   give better system performance. On the first boot, the operating system
+   automatically expands the file system to fill the card. This means that the
+   first boot will be substantially slower than subsequent boots.
+  </para>
+  <para>
+   The process of writing the card image onto &microsd; media is described in the
+   <link xlink:href="&dsc-sles;/html/SLES-rpi-quick/art-rpiquick.html">&rpiquick;</link>.
+  </para>
  </sect2>
- <sect2>
+
+ <sect2 xml:id="sec-aarch64-deprpi-usb">
+  <title>Installation from USB Media</title>
+  <para>
+   Some models of &rpi; allow to boot from USB mass storage devices.
+   This will then allow to deploy &productname; on &rpi; similar to server platforms.
+  </para>
+  <para>
+   Installation can be performed from a removable USB medium, such as a memory stick,
+   onto a &microsd; card in the machine's internal slot, or from a removable
+   USB medium onto another USB medium, such as a USB-connected hard disk.
+  </para>
+  <note>
+   <title>USB Bandwidth Limitations</title>
+   <para>
+    Note that the Ethernet controller on the &rpi; 3 is connected to the device's
+    on-board USB 2.0 bus.
+   </para>
+   <para>
+    Therefore an operating system running from a disk attached via USB
+    must share the total 480&nbsp;Mbps bandwidth of the USB 2.0 controller.
+    This will limit performance, and could significantly impact network performance.
+   </para>
+   <para>
+    This limitation does not apply to the &rpi; 4.
+   </para>
+  </note>
+  <para>
+   Newer models of &rpi; 3 with BCM2837 B0 silicon (silver instead of black chip),
+   including &rpi; 3 Model B+ and Compute Module 3+, allow to boot from USB-connected
+   storage devices by default.
+  </para>
+  <para>
+   On older models, such as &rpi; 3 Model B or Compute Module 3, USB boot can
+   be enabled by booting from a specially prepared &microsd; card once.
+   Please see <xref linkend="sec-aarch64-deprpi-otp"/> for instructions.
+  </para>
+ </sect2>
+
+ <sect2 xml:id="sec-aarch64-deprpi-net">
   <title>
-   Network Booting the &rpi;
+   Installation from Network
   </title>
   <para>
-   Due to the hardware's lack of on-board firmware, network-booting the &rpi;
-   using PXE is more complex than with more conventional computers.
+   Due to the hardware's lack of on-board firmware (<xref linkend="sec-aarch64-deprpi-boot"/>),
+   network-booting the &rpi; using PXE is more complex than with more conventional computers.
   </para>
   <para>
    The process of setting up a PXE boot server for &x86; and &arm; is described
```

Comment:
The URL needs a description: "...is described in `<citetitle>Network boot your Raspberry Pi</citetitle>`

Reply:
Why? I tested the PDF to look fine. And note that I renamed that section.

Reply:
Ah, note that the link is **not** to `-net` but to `-boot`. It's about the preceding "on-board firmware", not about the "network-booting", which is this very section.

Reply:
URLs always need a description of some kind so readers know when they're in the right place. URLs change and sometime Web browsers don't render deep links correctly. We use text descriptions rather than URL labels because URL labels render incorrectly in PDFs.

Reply:
Did you maybe comment on the wrong link? There is no URL here! It's an internal section reference that shows fine in the PDF as: `Section 3.6.1, "Boot Process"`